app/templates/settings/tier.hbs

```diff
@@ -3,16 +3,16 @@
         <h2 class="gh-canvas-title" data-test-screen-title>
             <LinkTo @route="settings">Settings</LinkTo>
             <span>{{svg-jar "arrow-right"}}</span>
-            <LinkTo @route="settings.products">Products</LinkTo>
+            <LinkTo @route="settings.tiers">Tiers</LinkTo>
```

Comment:
Is this a defunct template? I didn't think we had the word Product anywhere user-facing any more 🤔